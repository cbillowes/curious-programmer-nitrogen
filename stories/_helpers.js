import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page