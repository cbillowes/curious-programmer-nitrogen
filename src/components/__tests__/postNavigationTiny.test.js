import React from 'react'
import PostNavigationTiny from '../postNavigationTiny'
import { getSnapshot } from './_helpers'

describe(`Post navigation tiny`, () => {
  it(`should always render both posts`, () => {
    const previous = {
      slug: `/previous`,
      title: `Don't panic.`,
    }
    const next = {
      slug: `/next`,
      title: `Come to the dark side of the force.`,
    }
    const tiny = (
      <PostNavigationTiny
        previous={previous}
        next={next}
      />
    )
    const tree = getSnapshot(tiny)
    expect(tree).toMatchSnapshot()
  })
})
