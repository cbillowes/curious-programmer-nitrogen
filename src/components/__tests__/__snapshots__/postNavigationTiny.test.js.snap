// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post navigation tiny should always render both posts 1`] = `
<nav
  className="post navigation tiny"
>
  <aside
    className="previous"
  >
    <h1>
       
      <a
        href="/previous"
        onClick={[Function]}
        onMouseEnter={[Function]}
        title="Don't panic."
      >
        <span
          className="text"
        >
          Don't panic.
        </span>
      </a>
       
    </h1>
  </aside>
  <aside
    className="next"
  >
    <h1>
       
      <a
        href="/next"
        onClick={[Function]}
        onMouseEnter={[Function]}
        title="Come to the dark side of the force."
      >
        <span
          className="text"
        >
          Come to the dark side...
        </span>
      </a>
       
    </h1>
  </aside>
</nav>
`;
